### PR TITLE
Capstone Project Review

### PR DESCRIPTION
**Pull request by HunterX-7**
## In this Project, I:

- Followed GitHub flow.
- Use HTML&CSS and JavaScript Linters.
- Write meaningful commit messages in the most important parts of the process.
- Follow the following guidelines and implement these features:

  > Follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
  > Made the page almost identical to the original design with some little tweaks in the borders, text, and images.
  > Use BootStrap 5
  > Build two pages, the home page and the about page.
  > Made the page responsive to mobile and desktop screens.
  >  Add links to the Home page and the About page.
  > Add a mobile menu with a similar color to the original design that appears when you click the Hamburger icon and closes when you press the X button.
  > Add the "Feature Speakers Section" which is generated dynamically using javascript.
  > Create a nice README file with a description, the links to the [live](https://hunterx-7.github.io/Fashion-Week/) page deployment, and a [video](https://www.loom.com/share/fdde3a9996184ba59fba3e375171eec4) description of the project and gave credits and thanks to the original author [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

### CODE REVIEW

**Review by Chimwemwe-127001 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
### Hi @HunterX-7

Good job so far! :tada: 

There are some features that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

so close

## Highlight :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:

## Required Changes ♻️
- [x] Make sure you commit all feature implementation to your feature branches and not your main. Seeing that commits are irreversible, I ask that you may fix this method of committing, as it will not be suitable if you have coding partners. It will also make your commit history harder to interpret. 
- [x] I've noticed that your mobile version home page does not have a footer. As the requirements suggest, both pages contain the same navbar and footer. Kindly add the footer section to your mobile version home page.
- [x] As the requirements suggest, there's a link from the `Home` page to the `About` page and the opposite too that allows users to navigate between the two pages. When I navigate to the `About` page I don't see a `Home` link to navigate back to the `Home` page. Fix this both in mobile and desktop versions of the website.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by HunterX-7** [comment]
## In this Pull request, I made the required changes suggested:

- I will follow the suggestion of not merging before receiving a project review, I did it in this case because I forgot to deploy after adding the index files.
- Add the missing footer in the mobile version of the Home page.
Missing Footer in the mobile version
- Add the missing Home link in the nav bar of both pages.
Missing Home link in the nav bar

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
-# Fashion-Week-Panama
+# Fashion-Week
 This is a template for a Fashion and Clothing events page. The main objective is to showcase my skills in HTML/CSS, Bootstrap and JavaScript.
 ![screencapture-hunterx-7-github-io-Fashion-Week-2022-11-25-17_48_41](https://user-images.githubusercontent.com/90111138/204061316-5f7a9558-5934-42b1-a0d3-55537921c9db.png)
```
Comment (Owusu-Desmond) [inline README.md:5]: - [ OPTIONAL ] In your next project, please make sure that you use the updated [README TEMPLATE](https://github.com/microverseinc/readme-template) :pray:

**Review by Owusu-Desmond — APPROVED** [review]
Hi @HunterX-7 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**